Linux and Git Commands Cheatsheet

fonglh opened the pull request:
For VCS lesson

fonglh committed:
Add entry for Linux and Git commands.

fonglh committed:
Add cheatsheet of commonly used Linux and Git commands.

Fixes #9.

kxmbrian commented on chapters/5-linux-git.md:
```diff
@@ -0,0 +1,73 @@
+
+## Linux
+
+* `ls <directory name>`: List directory contents.
```
maybe caps 'l' and 's' in 'list' for consistency?

fonglh replied:
Ok, missed this one out.

kxmbrian commented on chapters/5-linux-git.md:
```diff
@@ -0,0 +1,73 @@
+In this cheatsheet, `<something>` indicates that `something` is a variable.
+You should replace it with the thing you want the command to act on.
+
+`<something ...>` means that the command can accept multiple arguments.
```
Would `<something> <...>` be clearer?
Otherwise `<directory name>` might give the impression that we are passing two arguments to cd.

fonglh replied:
I put the `...` within `< >` because that's how the man pages do it, although they use `[ ]`.

Maybe combine into `directory_name`?

kxmbrian replied:
that would work too

kxmbrian commented on chapters/5-linux-git.md:
```diff
@@ -0,0 +1,73 @@
+
+## Git
+
+Each command here is prefixed with `git`. They are subcommands which belong
```
Maybe clearer to say something like "Prefix each command with `git` to use it"

fonglh committed:
Make suggested improvements.

Emphasise L and S in "list" for consistency with the other commands.
Rename argument to directory_name.
Wording change about prefixing "git" to the commands.
